index.html

```diff
@@ -2349,6 +2349,94 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           </table>
         </div>
 
+        <p>
+          A <a>Thing Description</a> of a WoT producer may have multiple forms entries with, e.g., different 
+          protocol and/or content types declarations that a <a>Consumer</a> could possibly support. In that case 
```

Comment:
What should happen, when there are multiple entries for the same protocol and operation?
Is there a precedence?